dpnp/dpnp_iface.py

```diff
@@ -338,3 +339,13 @@ def get_normalized_queue_device(obj=None,
     if hasattr(dpt._device, 'normalize_queue_device'):
         return dpt._device.normalize_queue_device(sycl_queue=sycl_queue, device=device)
     return sycl_queue
+
+
+def isarray(obj):
+    """
+    Return True if:
```

Comment:
It'd be great to add more detailed description and to state what type of input `obj` is considered as array.